options/ansi/generic/file-io.cpp

```diff
@@ -331,6 +335,23 @@ int abstract_file::_write_back() {
 	return 0;
 }
 
+int abstract_file::_discard_buffer() {
+	if (_type == stream_type::file_like) {
```

Comment:
Is _type guaranteed to be initialized here?

Reply:
no, good point